about/docs/handbook/contributing.mdx

```diff
@@ -10,9 +10,17 @@ Want to contribute? Why not jump into our GitHub repo and:
 
 - [Sponsor Supabase](https://github.com/sponsors/supabase).
 - [Answer Discussions](https://github.com/supabase/supabase/discussions).
-- [Submit an issue](https://github.com/supabase/supabase/issue)
+- Submit an issue.
+    - [supabase/supabase](https://github.com/supabase/supabase/issues)
```

Comment:
thanks for the update @Isaiah-Hamilton !

I was actually thinking a separate section above the current list, something like:

```
<h1>How to contribute</h1>

Supabase has many [open source repos](https://github.com/orgs/supabase/repositories). Some of the more popular ones are:
  - [Supabase](link) - Our Dashboard, Websites, and example apps
  - [Supabase-js](link) - A clientside library written in Typescript
  - [Gotrue](link) - Our auth server, written in Go
  - [Realtime](link) - A server for streaming changes from Postgres, written in Elixir 
  - [browse all]((https://github.com/orgs/supabase/repositories))

Here are some ways you can contribute:
  - Sponsor Supabase
  - Start a discussion
  - Submit an issue to one of our [repos]((https://github.com/orgs/supabase/repositories))
... 
```